src/Form/EndpointForm.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\relaxed\Form\EndpointForm.
+ */
+
+namespace Drupal\relaxed\Form;
+
+use Drupal\Core\Entity\EntityForm;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\PluginFormInterface;
+
+/**
+ * Class EndpointForm.
+ *
+ * @package Drupal\relaxed\Form
```

Comment:
I don't think we use `@package` anywhere else in our source code. What does this do? Should we remove it, to at least be consistent? :)